.github/workflows/publish-draft-releases.yml

```diff
@@ -1169,4 +1170,96 @@ jobs:
           if [[ ${{ steps.get-runtime-version-base.outputs.runtime-ver-base }} == ${{ steps.get-runtime-version-current.outputs.runtime-ver-current }} ]]; then echo "::set-output name=match::true"
           fi
 
+  start-docker-image-tester:
+    runs-on: ubuntu-20.04
+    outputs:
+      runner-label: ${{ steps.start-self-hosted-runner.outputs.runner-label }}
+      aws-region: ${{ steps.start-self-hosted-runner.outputs.aws-region }}
+      aws-instance-id: ${{ steps.start-self-hosted-runner.outputs.aws-instance-id }}
+    steps:
+      -
+        id: start-self-hosted-runner
+        uses: audacious-network/aws-github-runner@v1.0.33
+        with:
+          mode: start
+          github-token: ${{ secrets.GH_SHR_TOKEN }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-instance-ssh-public-key: ${{ env.AWS_INSTANCE_SSH_PUBLIC_KEY }}
+          aws-region: ${{ env.AWS_REGION }}
+          aws-subnet-id: ${{ env.AWS_SUBNET_ID }}
+          aws-security-group-id: ${{ env.AWS_SECURITY_GROUP_ID }}
+          aws-instance-type: ${{ env.AWS_INSTANCE_TYPE }}  # 32 vcpu, 64gb ram, $1.392 hourly
+          aws-instance-root-volume-size: 64
+          aws-image-search-pattern: ${{ env.AWS_IMAGE_SEARCH_PATTERN }}
+          aws-image-search-owners: ${{ env.AWS_IMAGE_SEARCH_OWNERS }}  # canonical
+
+  docker-image-test:
+    needs:
+      - build-node-current
+      - start-docker-image-tester
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        uses: actions/checkout@v2
+      -
+        name: fetch manta
+        uses: actions/download-artifact@v2
+        with:
+          name: manta
+      -
+        name: build docker image
+        run: |
+          mv manta docker/manta
+          cd docker/
+          chmod +x manta
+          GIT_TAG=${{ github.event.release.tag_name }}
+          BINARY_NAME=manta
+          ln -s calamari.Dockerfile Dockerfile
+          docker build \
+            --build-arg PARA_BINARY_REF=${GIT_TAG} \
+            --build-arg PARA_BINARY_URL=manta \
+            --tag calamari   ./
```

Comment:
not that it matters (it really doesn't since this image doesn't get uploaded), but i guess a good tag that differentiates this container from the last one built, could be: `calamari:${GITHUB_SHA:0:7}`.